CREATE TABLE test (
	`ChromosomeNo` VARCHAR(300) NULL, 
	`Location` VARCHAR(300) NULL, 
	`Ref` VARCHAR(300) NULL, 
	`Alt` VARCHAR(300) NULL, 
	`GeneName` VARCHAR(300) NULL, 
	`chrMut` VARCHAR(300) NULL, 
	`pMut` VARCHAR(300) NULL, 
	`mutFrac` VARCHAR(300) NULL, 
	`TotReads` VARCHAR(300) NULL, 
	`VarReads` VARCHAR(300) NULL, 
	`Gene` VARCHAR(300) NULL, 
	`Feature` VARCHAR(300) NULL, 
	`Feature_type` VARCHAR(300) NULL, 
	`Consequence` VARCHAR(300) NULL, 
	`cDNA_position` VARCHAR(300) NULL, 
	`CDS_position` VARCHAR(300) NULL, 
	`Protein_position` VARCHAR(300) NULL, 
	`Amino_acids` VARCHAR(300) NULL, 
	`Codons` VARCHAR(300) NULL, 
	`Existing_variation` VARCHAR(300) NULL, 
	`IMPACT` VARCHAR(300) NULL, 
	`STRAND` VARCHAR(300) NULL, 
	`ENSP` VARCHAR(300) NULL, 
	`CANONICAL` VARCHAR(300) NULL, 
	`EXON` VARCHAR(300) NULL, 
	`INTRON` VARCHAR(300) NULL, 
	`GMAF` VARCHAR(300) NULL, 
	`CLIN_SIG` VARCHAR(300) NULL, 
	`SOMATIC` VARCHAR(300) NULL, 
	`PHENO` VARCHAR(300) NULL, 
	`FATHMM_pred` VARCHAR(300) NULL, 
	`FATHMM_score` VARCHAR(300) NULL, 
	`MetaLR_pred` VARCHAR(300) NULL, 
	`MetaLR_score` VARCHAR(300) NULL, 
	`MetaSVM_pred` VARCHAR(300) NULL, 
	`MetaSVM_score` VARCHAR(300) NULL, 
	`MutationAssessor_pred` VARCHAR(300) NULL, 
	`MutationAssessor_score` VARCHAR(300) NULL, 
	`PROVEAN_pred` VARCHAR(300) NULL, 
	`PROVEAN_score` VARCHAR(300) NULL, 
	`LRT_score` VARCHAR(300) NULL, 
	`LRT_pred` VARCHAR(300) NULL, 
	`fathmm.MKL_coding_pred` VARCHAR(300) NULL, 
	`fathmm.MKL_coding_score` VARCHAR(300) NULL, 
	`fathmm.MKL_coding_group` VARCHAR(300) NULL, 
	`MutationTaster_pred` VARCHAR(300) NULL, 
	`MutationTaster_score` VARCHAR(300) NULL, 
	`SIFT` VARCHAR(300) NULL, 
	`Polyphen2_HDIV_pred` VARCHAR(300) NULL, 
	`Polyphen2_HDIV_score` VARCHAR(300) NULL, 
	`Polyphen2_HVAR_pred` VARCHAR(300) NULL, 
	`Polyphen2_HVAR_score` VARCHAR(300) NULL, 
	`Uniprot_aapos_Polyphen2` VARCHAR(300) NULL, 
	`Uniprot_acc_Polyphen2` VARCHAR(300) NULL, 
	`FGSCat` VARCHAR(300) NULL
);
